src/dymaptic.GeoBlazor.Core/Components/Widgets/FormInput.cs

```diff
@@ -0,0 +1,184 @@
+﻿using Microsoft.AspNetCore.Components;
+
+
+namespace dymaptic.GeoBlazor.Core.Components.Widgets;
+
+/// <summary>
+///     Abstract base class for Input fields in a form element.
+/// </summary>
+public abstract class FormInput : MapComponent
+{
+    /// <summary>
+    ///     The type of form element input.
+    /// </summary>
+    public abstract string Type { get; }
+}
+
+/// <summary>
+///     The TextBoxInput class defines the desired user interface as a single-line text box.
```

Comment:
This is essentially a placeholder(non-functioning) at this time? and require a 3rd party library/application to function correct?